packages/phone-number-privacy/combiner/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@celo/phone-number-privacy-combiner",
```

Comment:
The only change here is the name and description, not sure why git didn't recognize it as a rename